README.md

```diff
@@ -49,13 +49,6 @@ If this feature list left you scratching your head, you can first read
 more about the [Vector Space Model] and [unsupervised document analysis]
 on Wikipedia.
 
-Support
```

Comment:
Why? Where users should place their questions?

Reply:
Section still there, I just moved it.